documentation.html

```diff
@@ -0,0 +1,460 @@
+                        style="width: 1024px; height: 260px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+                <h2>Horizontal Card</h2>
+                <div class="card-horizontal">
+                    <img class="card-horizontal-img" src="resources/avatar/avatar (5).png" alt="card-image" />
+                    <div class="card-horizontal-content">
+                        <h2 class="card-heading">Item Name</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nostrum fuga dicta aperiam vitae
+                            iste omnis molestiae, corporis magni.
+                        </p>
+                    </div>
+                </div>
+                <div class="code-block">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28171%2C184%2C195%2C0%29&t=one-light&wt=none&l=htmlmixed&ds=false&dsyoff=0px&dsblur=100px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=18px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card-horizontal%2522%253E%250A%2509%253Cimg%2520class%253D%2522card-horizontal-img%2522%2520src%253D%2522resources%252Favatar%252Favatar%2520%285%29.png%2522%2520alt%253D%2522card-image%2522%2520%252F%253E%250A%2509%253Cdiv%2520class%253D%2522card-horizontal-content%2522%253E%250A%2520%2520%2520%2520%2509%253Ch2%2520class%253D%2522card-heading%2522%253EItem%2520Name%253C%252Fh2%253E%250A%2520%2520%2520%2520%2509%253Cp%253ELorem%2520ipsum%2520dolor%2520sit%2520amet%2520consectetur%252C%2520adipisicing%2520elit.%253C%252Fp%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                        style="width: 900px; height: 260px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+                <h2>Cart Card</h2>
+                <div class="card">
+                    <img class="card-wishlist" src="resources/icons/heart_.png" />
```

Comment:
wishlist icon should act as a button, you can try image or icon inside a button element.